task_38.py

```diff
@@ -0,0 +1,62 @@
+# Дополнить телефонный справочник возможностью изменения и удаления данных. 
+# Пользователь также может ввести имя или фамилию, 
+# и Вы должны реализовать функционал для изменения и удаления данных
+
+def show_info():
+    with open( 'phone_book.txt' , 'r' ) as file:
+        book = file.read().split('\n')
+    print(book)
+
+def new_string():
+    with open( 'phone_book.txt' , 'a') as file:
+        file.write(input('Введите новую строку: '))
+
+def find_info():
+    with open( 'phone_book.txt' , 'r') as file:
+        book = file.read().split('\n')
+        t = input('Что вы хотите найти?   ')
+        for i in book:
+            if t in i:
+                print(i)
```

Comment:
Читать методом `read` не обязательно `file` является итератором
```suggestion
        t = input('Что вы хотите найти?   ')
        for i in file:
            if t in i:
                print(i)
```